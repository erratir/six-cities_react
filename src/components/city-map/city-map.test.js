import React from 'react';
import renderer from 'react-test-renderer';

import CityMap from './city-map.jsx';

const offers = [
  {
    "city": {
      "name": `Kukuevo`,
      "location": {
        "latitude": 50.846557,
        "longitude": 4.351697,
        "zoom": 13,
      },
    },
  },
  {
    "city": {
      "name": `New Vasuki`,
      "location": {
        "latitude": 50.938361,
        "longitude": 6.959974,
        "zoom": 13,
      },
    },
  },
  {
    "city": {
      "name": `Мухосранск`,
      "location": {
        "latitude": 51.225402,
        "longitude": 6.776314,
        "zoom": 13,
      },
    },
  },
];

describe(`CityMap component`, () => {
  it(`render correctly`, () => {
    const tree = renderer
      .create(<CityMap offers={offers}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
