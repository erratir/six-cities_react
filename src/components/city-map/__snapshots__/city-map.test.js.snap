// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityMap component render correctly 1`] = `
<section
  className="cities__map map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
